cpmpy/transformations/linearize.py

```diff
@@ -367,18 +489,20 @@ def only_positive_bv(lst_of_expr, csemap=None):
 
     return newlist
 
-def only_positive_bv_wsum(expr):
+def only_positive_bv_wsum(expr: Expression) -> Expression:
     """
-        Replaces a var/sum/wsum expression containing :class:`~cpmpy.expressions.variables.NegBoolView` with an equivalent expression 
-        using only :class:`~cpmpy.expressions.variables.BoolVar`. 
-
-        It might add a constant term to the expression, if you want the constant separately, use :func:`only_positive_bv_wsum_const`.
-        
-        Arguments:
-        - `cpm_expr`: linear expression (sum, wsum, var)
-        
-        Returns tuple of:
-        - `pos_expr`: linear expression (sum, wsum, var) without NegBoolView
+    Replaces a var/sum/wsum expression containing :class:`~cpmpy.expressions.variables.NegBoolView`
+    with an equivalent expression using only :class:`~cpmpy.expressions.variables.BoolVar`.
+
+    It might add a constant term to the expression. If you want the constant separately,
+    use :func:`only_positive_bv_wsum_const`.
+
```

Comment:
I'm wondering if this doc needs an example.

"E.g., the weighed sum `3*~x + 4*y` is transformed to `3 + -3*x + 4*y`. If you want the constant 3 to be separate, use `only_positive_bv_wsum_const` instead."